support-frontend/app/admin/ServersideAbTest.scala

```diff
@@ -1,23 +1,31 @@
 package admin
-import cookies.ServersideAbTestCookie
-import play.api.mvc.RequestHeader
+import admin.ServersideAbTest.Participation
+import io.circe.Encoder
 
 import scala.util.Random
 
 object ServersideAbTest {
   // Serverside A/B tests currently only support a single concurrent test
   // running to 100% audience with a 50%/50% split
-  def getParticipation(implicit request: RequestHeader): Participation = {
-    ServersideAbTestCookie.get.flatMap(_.value match {
-      case "Control" => Some(Control)
-      case "Variant" => Some(Variant)
-      case _ => None
-    }).getOrElse(computeParticipation)
-  }
 
-  private def computeParticipation: Participation = if (Random.nextBoolean) Control else Variant
+  def generateParticipations(testNames: List[String]): Map[String, Participation] =
+    testNames.map(_ -> computeParticipation).toMap
+
+  def computeParticipation: Participation = if (Random.nextBoolean) Control else Variant
```

Comment:
So this makes our server-side A/B tests non-sticky? (That's fine, and actually gives us more options and is better for GDPR) it'd just be good to socialise this amongst dev teams so everyone is aware.

Reply:
Correct - and it's how we've done server-side tests previously.
If we want to use MVT then we have to worry about having a unique seed value as we do with client-side tests